Ahorn/lang/en_gb.lang

```diff
@@ -441,11 +441,32 @@ placements.entities.CommunalHelper/SJ/AirTimeMusicController.tooltips.musicParam
 placements.entities.CommunalHelper/SJ/WormholeBooster.tooltips.deathColor=The color to use for the booster when it becomes deadly.
 placements.entities.CommunalHelper/SJ/WormholeBooster.tooltips.instantCamera=Whether the camera should transition instantly or not.
 
+# -- [Strawberry Jam] Flag Breaker Box --
+placements.entities.CommunalHelper/SJ/FlagBreakerBox.tooltips.flag=The flag to control.
+placements.entities.CommunalHelper/SJ/FlagBreakerBox.tooltips.aliveState=The value to set the flag to when the box is alive (sets to the oposite when destroyed).
+placements.entities.CommunalHelper/SJ/FlagBreakerBox.tooltips.music=The music to play when the box is broken.
+placements.entities.CommunalHelper/SJ/FlagBreakerBox.tooltips.flipX=Whether or not to flip the sprite across the X axis.
+placements.entities.CommunalHelper/SJ/FlagBreakerBox.tooltips.music_progress=The progress to set the music to when the box is broken.
+placements.entities.CommunalHelper/SJ/FlagBreakerBox.tooltips.music_session=Whether or not to store the music changes in the session.
+
+# -- [Strawberry Jam] Momentum Block --
+placements.entities.CommunalHelper/SJ/MomentumBlock.tooltips.speed=The speed magnitude to impart on the player.
+placements.entities.CommunalHelper/SJ/MomentumBlock.tooltips.speedFlagged=The speed magnitude to impart on the player when the given flag is set.
+placements.entities.CommunalHelper/SJ/MomentumBlock.tooltips.direction=The angle of the momentum imparted on the player.
+placements.entities.CommunalHelper/SJ/MomentumBlock.tooltips.directionFlagged=The angle of the momentum imparted on the player when the given flag is set.
+placements.entities.CommunalHelper/SJ/MomentumBlock.tooltips.flag=When this flag is set, the speedFlagged and directionFlagged attributes will be used instead of speed/direction.\nLeave blank for speed/direction to always be used.
+
```

Comment:
missing tooltip entries for `DashZipMover.jl`

Reply:
That's correct. DashZipMover doesn't have any properties.